examples/particles/js/editor/controls.js

```diff
@@ -46,7 +45,6 @@
             emitter.name = "emitter" + me.utils.createGUID();
             emitter.z = 10;
```

Comment:
I don't know what you mean by "below", but all references to emitter.z just won't work. They should all be emitter.pos.z... I didn't look through the example too hard when I made the change. Was mostly testing it out in my own game.